integreat_cms/cms/templates/tutorials/page_access_statistics.html

```diff
@@ -0,0 +1,38 @@
+{% extends "../generic_tutorial_dialog.html" %}
+{% load i18n %}
+{% load static %}
+{% block content %}
+    <!-- Pop-Up Header -->
+    <div class="flex items-center bg-water-500 w-full rounded h-12">
+        <h2 class="flex items-center font-semibold leading-normal pl-2 text-black text-xl">
+            {% translate "Important information about page access" %}
+        </h2>
+    </div>
+    <!-- Pop-Up Content -->
+    <div class="px-4">
```

Comment:
In the design there are 28px to the left and right, and 20px to the top and bottom :) 
```suggestion
    <div class="px-7 pt-5 pb-4">
```